Contest_API.md

```diff
@@ -900,6 +900,8 @@ JSON elements of problem objects:
 | time\_limit       | number  | no        | no        | Time limit in seconds per test data set (i.e. per single run). Should be an integer multiple of `0.001`.
 | test\_data\_count | integer | yes       | no        | Number of test data sets.
 | max_score         | number  | no        | no        | Maximum expected score, although teams may score higher in some cases. Typically used to indicate scoreboard cell color in scoring contests. Required iff contest:scoreboard_type is `score`.
+| problem_package   | array of FILE  | no | yes    | [Problem package](https://www.kattis.com/problem-package-format/). Expected mime type is application/zip. Only exactly one archive is allowed. Not expected to actually contain href for package during the contest, but used for configuration and archiving.
```

Comment:
Did we ever decide/clarify whether a problem package zip contains the files inside a directory of the problem name or in the root of the zip? I guess that's something that has to be clarified in the problem-package-format repo, but I think it's relevant here, as this is meant for potentially build automated tooling on top of that file format.

Reply:
> Did we ever decide/clarify whether a problem package zip contains the files inside a directory of the problem name or in the root of the zip?

I don't remember.

I think we should say that the directory must be included.

> I guess that's something that has to be clarified in the problem-package-format repo, 

Yes.

> but I think it's relevant here, as this is meant for potentially build automated tooling on top of that file format.

Sure it's relevant, but the correct place to document all the details about the problem package format is in the problem package format specification. 

In summary, I don't think we should change the current wording here, but we should update the problem package format to clarify this.